azure-pipelines.yml

```diff
@@ -5,14 +5,15 @@ jobs:
     vmImage: 'vs2017-win2016'
   steps:
   - powershell: |
-      $buildId = $env:BUILD_BUILDNUMBER.PadLeft(7, '0');
-      $versionSuffixPR = "ci-$buildId-pr$($env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER)";
-      $branchName = "$env:BUILD_SOURCEBRANCHNAME".Replace("_","");
-      $versionSuffixBRANCH = "$branchName-$buildId";
-      $isTag = "$env:BUILD_SOURCEBRANCH".StartsWith('refs/tags/');
-      $isPR = "$env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER" -ne ""
-      $versionSuffix = if ($isTag) { "" } else { if ($isPR) { $versionSuffixPR } else { $versionSuffixBRANCH } };
-      Write-Host "##vso[task.setvariable variable=VersionSuffix]$versionSuffix";
+      $buildId = $env:BUILD_BUILDNUMBER.PadLeft(7, '0')
+      if ("$env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER" -ne "") {
+        $meta = "$buildId-pr$($env:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER)"
+      } else {
+        $branchName = "$env:BUILD_SOURCEBRANCHNAME".Replace("_","");
+        $meta = "$buildId-$branchName";
+      }
```

Comment:
Maybe I'm missing something but it seems like `$meta` is only being assigned, and its value is not actually used anywhere.